tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/AdoNetClientInstrumentMethodsAttribute.cs

```diff
@@ -16,7 +16,7 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet
     /// as string. Avoid changing the name and/or namespace of this class.
     /// </remarks>
     [AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
-    internal sealed class AdoNetClientInstrumentMethodsAttribute : Attribute
```

Comment:
Did you need to remove `sealed`? I don't see any new types inheriting from `AdoNetClientInstrumentMethodsAttribute`.

Reply:
I don't know where my previous comment went. But basically, you are right. I tried to add a base class to handle Reader overloads, without success. I reverted it.